policyengine_us/parameters/gov/states/az/tax/income/rebates/amount.yaml

```diff
@@ -0,0 +1,26 @@
+description: Arizona provides the following amounts of Tax Rebate, depending on child age.  
+
+metadata:
+  unit: currency-USD
+  period: year
+  label: Arizona one-time families tax rebate per dependent 
+  type: single_amount
+  threshold_unit: year
+  amount_unit: currency-USD
+  reference:
+  - title: Arizona Families Tax Rebate
+    href: https://211arizona.org/wp-content/uploads/2023/11/Arizona-Families-Tax-Rebate.pdf#page=1
+  - title: Arizona SENATE BILL 1734, Section 3(A)
+    href: https://www.azleg.gov/legtext/56leg/1R/bills/SB1734S.pdf#page=3 
+  - title: ARIZONA STATE SENATE Fifty-Sixth Legislature, First Regular Session
```

Comment:
from this reference I see:
```between October 15, 2023, and November 15, 2023```
For the dates when the amount is paid out, 


And the age requirement applies in 2021 so we need to check for the age in 2021 (depending on whether it applies just in 2023 or also in following years we could reduce the age by 2 and exclude children under 2)